README.md

```diff
@@ -1 +1,234 @@
-# react-project
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <h3><b>Covid Report</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+    - [🚀 Live Demo](#live-demo)
+    - [🚀 Project Presentation](#project-presentation)
+- [💻 Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 Attribution](#attribution)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Covid Report <a name="about-project"></a>
+
+Stay informed about the latest COVID-19 statistics with our COVID-19 Data App. Our user-friendly interface and interactive visuals provide real-time data on cases, recoveries, and fatalities. Filter and search for specific information, explore region-specific details, and make informed decisions based on accurate, reliable data sourced from trusted health organizations. Download the app today and stay connected to the latest COVID-19 updates to navigate these challenging times with confidence.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>React</summary>
+  <ul>
+    <li><a href="https://react.dev/">React.js</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Redux</summary>
+  <ul>
+    <li><a href="https://redux.js.org/">Redux-toolkit</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Node js</summary>
+  <ul>
+    <li><a href="https://nodejs.org/en">Redux-toolkit</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Axios</summary>
+  <ul>
+    <li><a href="https://axios-http.com/">Axios</a></li>
+  </ul>
+</details>
+ <details>
+  <summary>React Testing Library</summary>
+  <ul>
+    <li><a href="https://testing-library.com/docs/react-testing-library/intro/">React Bootstrap</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+## 🛠 Key Features <a name="key-features"></a>
+
+### Features <a name="key-features"></a>
+
+**Home Page:Display report in Australia**
+<br />
+
+**Details Page:Display covid details**
+
+<p align="right">
+(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo](https://australia-covid-report.netlify.app/)
+
+## 🚀 Project Presentation <a name="project-presentation"></a>
+
+- [Project Presentation ](https://www.loom.com/share/6287d1be16ae4978b98aa5af5f86d754?sid=ae02ede1-d59f-4db4-b345-af921d5ec416)
```

Comment:
Good job recording your video but you didn't tell us about a feature you are most proud and also your video is only 2mins long and is too brief  it would be nice if you did so by recording another by following the below requirements:
Video presentation

Provide feedback about the video presentation in the code review summary.

The student recorded a video presenting the features of the project.
The maximum length of the video is 5 minutes.
The video contains a clear and objective description of the project in professional English. The topic and purpose of the project are made clear.
The video contains a demo of the project, highlighting all the project features.
The video contains a highlight about a piece of the code the student finds interesting or is proud of making.


Note: It's recommended that students use [Loom](https://www.loom.com/) to record to video but they can use any tool they want as long as they share an online link of the uploaded video.